ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```
@@ -338,22 +338,26 @@ rm /tmp/pull-secret
+  if [[ "${OCP_ARCH}" == "arm64" ]]; then
+    ami_arch_key="aarch64"
+  fi
   if version_le "${ocp_version}" "4.9"; then
     # 4.9 and below
     curl -sL https://raw.githubusercontent.com/openshift/installer/release-${ocp_major_version}.${ocp_minor_version}/data/data/rhcos.json -o /tmp/ami.json
     CONTROL_PLANE_AMI=$(jq --arg r $aws_source_region -r '.amis[$r].hvm' /tmp/ami.json)
   elif version_le "${ocp_version}" "4.21"; then
     # 4.10 to 4.21
     curl -sL https://raw.githubusercontent.com/openshift/installer/release-${ocp_major_version}.${ocp_minor_version}/data/data/coreos/rhcos.json -o /tmp/ami.json
-    CONTROL_PLANE_AMI=$(jq --arg r $aws_source_region -r '.architectures.x86_64.images.aws.regions[$r].image' /tmp/ami.json)
+    CONTROL_PLANE_AMI=$(jq --arg r $aws_source_region --arg arch $ami_arch_key -r '.architectures[$arch].images.aws.regions[$r].image' /tmp/ami.json)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)